Jenkinsfile

```diff
@@ -77,19 +57,21 @@ pipeline {
         not { expression { isPr } }
       }
       steps {
-        script { 
-          env.REGISTRY_PUSH = (params.RELEASE_BUILD) ? 'all' : 'internal'
-          env.SERVICE_VERSION = sbtbuild.getServiceVersion()
-          // set the SERVICE_SHA to the current head because it might not be the same as env.GIT_COMMIT
```

Comment:
Since we have removed the git commands above, we can just use env.GIT_COMMIT for the service SHA.